crates/nargo_cli/tests/execution_success/slices/src/main.nr

```diff
@@ -48,6 +49,7 @@ fn main(x : Field, y : pub Field) {
     regression_2083();
     // The parameters to this function must come from witness values (inputs to main)
     regression_merge_slices(x, y);
+    regression_dynamic_slice_index(x - 1, x - 4);
```

Comment:
Can you split this into a separate test?

Reply:
done